CADETProcess/optimization/optimizationProblem.py

```diff
@@ -2187,7 +2187,7 @@ def evaluate_linear_constraints(self, x):
         """
         values = np.array(x, ndmin=1)
 
-        return self.A.dot(values) - self.b
+        return self.A_transformed.dot(values) - self.b_transformed
```

Comment:
Here, I think we need to be careful. This function should also work with the `untransform=True` flag which creates x from x_untransformed and consequently evaluates `A`